service/fexservice/consumer.py

```diff
@@ -1,20 +1,12 @@
 import dataset
 from dynaconf import settings, Validator
 from github import Github
-from github.GithubException import BadAttributeException  # TODO
-from github.GithubException import BadUserAgentException  # TODO
-from github.GithubException import IncompletableObject  # TODO
-from github.GithubException import (BadCredentialsException,
-                                    RateLimitExceededException,
-                                    TwoFactorException, UnknownObjectException)
-from requests.exceptions import ConnectionError  # Error de conexão
-from requests.exceptions import ReadTimeout  # Error de conexão
-from sqlalchemy.exc import OperationalError
 
+
+from fexservice.Exception import ConsumerCritical, ConsumerWarning
```

Comment:
```suggestion
from fexservice.exception import ConsumerCritical, ConsumerWarning
```

Seria bom renomear o arquivo para `exception.py` para ficar tudo em minusculo :) [veja mais aqui](https://www.python.org/dev/peps/pep-0008/#package-and-module-names)